### PR TITLE
Revert "Child 1"

### PR DESCRIPTION
**Pull request by jaxinye**
Reverts jaxinye/UMSITest#1

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +0,0 @@
-[submodule "Python-100-Days"]
-	path = Python-100-Days
```
Comment (gitprojecttest1) [inline .gitmodules:2]: revert

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +0,0 @@
-[submodule "Python-100-Days"]
```
Comment (gitprojecttest1) [inline .gitmodules:1]: ffff....

**Review by ghost — DISMISSED** [review]
ccccc

File: .gitmodules (hunk 2 above)
Comment (jaxinye) [reply .gitmodules:1]: jaxinye

**Comment by jaxinye** [comment]
ggg

**Review by ghost — CHANGES_REQUESTED** [review]
comment

File: child1.py [hunk 3]
```diff
@@ -1,80 +0,0 @@
...
-import datetime
-
-def save_html(time, html):
-    with open(time + '.html', 'wb') as outfile:
-        outfile.write(html.read())
-        print('html saved')
-
-#create list of repositories where each repository is a dictionary
-def scrape():
-    html = urlopen("https://github.com/trending")
```
Comment (gitprojecttest1) [inline child1.py:19]: comment1

**Review by ghost — CHANGES_REQUESTED** [review]
comment2

**Pull request closed without merging**